src/app/prescription/factories/factory.go

```diff
@@ -0,0 +1,15 @@
+package prescription_factories
+
+import (
+	"clinic-api/src/app/prescription/services"
+	prescription_handlers "clinic-api/src/app/prescription/handlers"
+	prescription_repositories "clinic-api/src/app/prescription/repositories"
+
+	"gorm.io/gorm"
+)
+
+func Factory(conn *gorm.DB) prescription_handlers.Handler {
+	repo := prescription_repositories.NewMySQLRepository(conn)
+	serv := services.NewService(repo)
+	return *prescription_handlers.NewHandler(serv)
+}
```

Comment:
Please add a line feed by re-save on this file.

Reply:
done https://github.com/ALTERA-CAPSTON-41/backend/pull/62/commits/677b56860bcfb48f4cf17ca0793de6b9fb941637